hack/kube/overlays/dev-am/enduro-am.yaml

```diff
@@ -81,6 +81,11 @@ spec:
                 secretKeyRef:
                   name: enduro-am-secret
                   key: api_key
+            - name: ENDURO_AM_TRANSFERSOURCEPATH
+              valueFrom:
+                secretKeyRef:
+                  name: enduro-am-secret
+                  key: transfer_source_path
```

Comment:
If you are planning to include it in the secret, please, add it in the documentation:

https://github.com/artefactual-sdps/enduro/blob/main/docs/src/dev-manual/archivematica.md?plain=1#L78

I also think it's okay not to consider it in the secret, as it's done with some of the other config options.

Reply:
Okay, I'll add it to the docs.  

I don't think it needs to be a secret, except that I want to be able to change the transfer source path in dev without modifying `enduro.toml` (or any other files that might get commited) and there's no other way to do that AFAIK.